proposed/2021/DotnetNugetTrustedSigners.md

```diff
@@ -0,0 +1,196 @@
+
+USAGE:
+    dotnet nuget trusted-signers trust-package [OPTIONS] <name> <package>...
+
+OPTIONS:
+        --allow-untrusted-root          Specifies if the certificate for the trusted signer should be allowed to chain to an untrusted root.
+        --author                        Specifies that the author signature of the package(s) should be trusted. This option is mutually exclusive with --repository.
+        --repository                    Specifies that the repository signature or countersignature of the package(s) should be trusted. This option is mutually exclusive with --author.
+        --owners                        Semi-colon separated list of trusted owners to further restrict the trust of a repository. Only valid when using the --repository option.
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+```
+
+#### `dotnet nuget trusted-signers trust-certificate`
+
+```
+Adds a trusted signer with the given name to the config, based on a certificate fingerprint.
+
+Note: If a trusted signer with the given name already exists, the certificate item will be added to that signer. Otherwise a trusted author will be created with a certificate item from given certificate information.
+
+USAGE:
+    dotnet nuget trusted-signers trust-certificate [OPTIONS] <name> <fingerprint>
+
+OPTIONS:
+        --allow-untrusted-root          Specifies if the certificate for the trusted signer should be allowed to chain to an untrusted root.
+        --algorithm                     Specifies the hash algorithm used to calculate the certificate fingerprint. Defaults to SHA256. Values supported are SHA256, SHA384 and SHA512
+    -h, --help                          Prints help information.
+    -v, --verbosity <LEVEL>             Set the MSBuild verbosity level. Allowed values are q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic].
+        --interactive                   Allows command to stop and wait for user input or action.
+        --configfile                    The NuGet configuration file. If specified, only the settings from this file will be used. If not specified, the hierarchy of configuration files from the current directory will be used.
+```
+
+#### `dotnet nuget trusted-signers trust-source`
+
+```
+Adds a trusted signer based on a given source.
+
+If only `<name>` is provided, and it's an existing configured source, this command will add a trusted signer with the same name as that source to the trusted list.
```

Comment:
What do you think of this phrasing?

```suggestion
If only `<name>` is provided, the existing package source in your NuGet configuration files with the same name will be added to the trusted list.
```

(This sentence is really hard! Maybe pull in a copywriter haha)